packages/react/src/utils/positioner/Popper.tsx

```diff
@@ -186,6 +187,8 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
           placement: computedPlacement,
           scheduleUpdate,
         })
+      : children && rest && Object.keys(rest).length > 0
+      ? React.cloneElement(React.Children.only(children as React.ReactElement<any>), rest)
```

Comment:
Do we still need this change?

Reply:
I will separate this to different PR, but it will be required in the future